tests/invoke/invoke_kernel_params.cpp

```diff
@@ -148,10 +151,12 @@ class TEST_NAME : public util::test_base {
 
           // access the output
           auto acc_pass =
-              buf_result
-                  .template get_access<sycl::access_mode::write,
-                                       sycl::target::device>(
-                      cgh);
+<<<<<<< HEAD
```

Comment:
Merge conflict marker

Reply:
Thanks for the pay attention, I've removed this in this commit: f6cd54a48b76cbfb3cd2ad739c05ed42d43dcd3c